web/themes/joinup/prototype/sass/_drupal-hacks.sass

```diff
@@ -108,7 +108,7 @@ textarea
 
 .comment-item
   ul.contextual-links
-    z-index: 3
+    z-index: $z-index-highlight + 2
```

Comment:
These could really be maps in sass https://webdesign.tutsplus.com/tutorials/an-introduction-to-sass-maps-usage-and-examples--cms-22184